Jenkinsfile

```diff
@@ -7,6 +7,7 @@ devToolsProject.run(
   setup: { data ->
     Object venv = virtualenv.create('python3.8')
     venv.run('pip install -r requirements-dev.txt')
+    venv.run('ansible-galaxy install -f -r requirements.yml')
```

Comment:
What do the changes in the `Jenkinsfile` have to do with adding a dependency in `ableton.pkg_mgr_path`? 😅

Reply:
Because when we add that dependency, running `molecule test` or anything else that uses this role will apply the role dependencies first. We need to make sure the dependencies exist, hence the need to add the `requirements.yml` and actually install them on CI.